app/code/community/Bolt/Boltpay/Block/Form.php

```diff
@@ -16,15 +16,18 @@
  */
 
 /**
- * This sets the hidden Bolt payment form
- * that is used in callback to save the order
- *
- * @deprecated Getting order data is done through the Bolt Fetch API call
+ * Effects how Bolt is displayed when listed as a choice amongst other payment options
```

Comment:
Since we're updating comment, maybe "Affects" is a better word to use here?

Reply:
@adentong 
Not maybe.  Definitely.  One is a verb, the other is a noun.  Feel free to speak directly.